src/modules/flow/iio/nodes.c

```diff
@@ -25,15 +25,39 @@
 
 #include <sol-iio.h>
 
-struct gyroscope_data {
+enum iiodevice_types {
+    IIO_GYROSCOPE=1,
+    IIO_MAGNETOMETER=2,
+    IIO_THERMOMETER=3,
+    IIO_PRESSURE_SENSOR=4,
+    IIO_COLOR=5,
+    IIO_ACCELEROMETER=6,
+    IIO_HUMIDITY_SENSOR=7,
+    IIO_ADC=8,
+    IIO_LIGHT_SENSOR=9,
+IIO_PROXIMITY=10,
+};
```

Comment:
I would like to use these enum items to re-use these common functions, I just uploaded a new version, please help to review